### PR TITLE
🌐 Updates "New Application" microcopy

### PR DESCRIPTION
**Pull request by josemigallas**
**What this PR does / why we need it**:

Updates strings and texts in Applications index and New Application form

**Which issue(s) this PR fixes** 

[THREESCALE-7022: [UI Strings] Add 'Create new Application' flow to Audience > Applications index](https://issues.redhat.com/browse/THREESCALE-7022)

**Verification** 

Screenshot 2021-11-10 at 12 26 51
Screenshot 2021-11-10 at 12 30 11
Screenshot 2021-11-10 at 12 28 20
Screenshot 2021-11-10 at 12 28 11
Screenshot 2021-11-10 at 12 27 59
Screenshot 2021-11-10 at 12 27 50
Screenshot 2021-11-10 at 12 27 25
Screenshot 2021-11-10 at 12 27 17
Screenshot 2021-11-10 at 12 27 08

### CODE REVIEW

**Review by akostadinov — APPROVED** [review]
New wording sounds clearer to me. Thanks.

**Comment by dfennessy** [comment]
LGTM!

**Review by dfennessy — DISMISSED** [review]

**Comment by josemigallas** [comment]
@dfennessy Sorry I should've added some screenshots, there was 1 string left unfixed.

**Review by akostadinov — APPROVED** [review]

**Review by damianpm — COMMENTED** [review]
lgtm, but please update tests

**Review by akostadinov — DISMISSED** [review]

File: app/views/api/applications/new.html.slim [hunk 1]
```diff
@@ -1,4 +1,4 @@
-h2 New Application
+h2 New application
```
Comment (dfennessy) [inline app/views/api/applications/new.html.slim:1]: Should this be "Create application"? See comments in the GDoc.

File: app/views/provider/admin/applications/new.html.slim [hunk 2]
```diff
@@ -1,4 +1,4 @@
-h2 New Application
+h2 New application
```
Comment (dfennessy) [inline app/views/provider/admin/applications/new.html.slim:1]: Same as above comment. Re: Create application

**Review by dfennessy — COMMENTED** [review]
Just a couple of comments.

**Review by dfennessy — APPROVED** [review]
LGTM!

**Pull request merged**